tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/BasicCasesTest.java

```diff
@@ -87,7 +87,7 @@ public void testAssertSequenceTypes() throws BallerinaTestException {
         AssertionUtils.assertForTestFailures(output, "assertion failure for sequence data types");
     }
 
-    @Test
+    @Test(enabled = false)
```

Comment:
Why is this being disabled?

Reply:
Enabled with https://github.com/ballerina-platform/ballerina-lang/pull/35567